src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml

```diff
@@ -19,7 +21,9 @@
 		<Setter Property="FontFamily"
 				Value="{StaticResource FontFamily01}" />
 		<Setter Property="Padding"
-				Value="20" />
+				Value="20,9.5,20,7.5" />
```

Comment:
How does it look with integer instead of decimal?

Reply:
It look good but I can round up.

Reply:
Let's go with that for now

Reply:
Let's go with that for now